### ISSUE TITLE
Implement language switches for titles of plays

### ISSUE THREAD

**Issue by lehkost** [dracor-org/dracor-frontend#135]
After implementing https://github.com/dracor-org/dracor-api/issues/118 we could start implementing a language switch for titles of plays. Language codes should be delievered by the API and could then result in a switch button on corpus overview pages and pages for single plays.

**Comment by cmil** [comment]
If my assumption in https://github.com/dracor-org/dracor-api/issues/118#issuecomment-761604499 is right then I would assume that this language switch would toggle between original language and English. Thus it would only make sense for non-English corpora and only if a non-English corpus has a significant number of translated titles and names.

**Comment by lehkost** [comment]
In the corpus overviews, I would add a small "EN" button in both the "Authors" and "Title" column headers to switch to English author names and play titles. This way, it would be visually clear, that only these two columns are affected.

It's harder to decide where to place that "EN" button in the single-play view. On these pages we should also offer a switch to change to display English character names if available (for both the network and the character list to the right).

**Comment by cmil** [comment]
@lehkost is it really necessary to toggle the language on page or even column basis. It would be easier to implement this as a side wide setting with a single toggle for instance in the top navigation (maybe with a little gear icon and a menu item "Prefer English names and titles"). I think it would be more clear that this does not translate the site consistently but provides English titles only where they are available. Also, a central toggle would scale better should we decide to add more languages.

**Comment by lehkost** [comment]
Yes, that's true – a side-wide toggle is better. We could probably describe it as "Show metadata of plays in English if available", and could detail it, if that helps to understand what the function does, to apply to "authors, titles, character names".
